terraform/azure/ha-availability-set-new-vnet/README.md

```diff
@@ -10,19 +10,41 @@ As part of the deployment the following resources are created:
 See the [Check Point CloudGuard IaaS High Availability for Azure Administration Guide](https://sc1.checkpoint.com/documents/IaaS/WebAdminGuides/EN/CP_CloudGuard_IaaS_HighAvailability_for_Azure/Content/Topics/Check_Point_CloudGuard_IaaS_High_Availability_for_Azure.htm) for additional information
 
 This solution uses the following modules:
-- /cgi-terraform/azure/modules/common - used for creating a resource group and defining common variables.
-- /cgi-terraform/azure/modules/vnet - used for creating new virtual network and subnets.
-- /cgi-terraform/azure/modules/network-security-group - used for creating new network security groups and rules.
+- /terraform/azure/modules/common - used for creating a resource group and defining common variables.
+- /terraform/azure/modules/vnet - used for creating new virtual network and subnets.
+- /terraform/azure/modules/network-security-group - used for creating new network security groups and rules.
 
 
 ## Configurations
 - Install and configure Terraform to provision Azure resources: [Configure Terraform for Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/terraform-install-configure)
-- In order to use ssh connection to VMs, add a public key to the /cgi-terraform/azure/ha-availability-set/azure_public_key file
+- In order to use ssh connection to VMs, it is **required** to add a public key to the /terraform/azure/ha-availability-set/azure_public_key file
+<br>In case there is no need in the ssh key usage, next lines in the main.tf file need to be deleted or commented:
 
+        ssh_keys {
+          path = "/home/notused/.ssh/authorized_keys"
+          key_data = file("${path.module}/azure_public_key")
+        }
 ## Usage
-- Create a Service Principal with the following permissions to the Azure subscription. 
-This service principal will be used by Terraform in order to deploy the solution.
-- Fill all variables in the /cgi-terraform/azure/ha-availability-set/terraform.tfvars file with proper values (see below for variables descriptions).
+- Choose the preferred login method to the Azure in order to deploy the solution:
+    <br>1. Using Service Principal:
+    - Create a [Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) (or use the existing one) 
+    - Grant the Service Principal at least "**Owner**" permissions to the Azure subscription<br>
+
+    <br>2. Using **az** commands from a command-line:
+    - Run  **az login** command 
+    - Sign in with your account credentials in the browser
+    - [Accept Azure Marketplace image terms](https://docs.microsoft.com/en-us/cli/azure/vm/image/terms?view=azure-cli-latest) by running:
+     <br>**az vm image terms accept --urn publisher:offer:sku:version**, where:
+        - publisher = checkpoint;
+        - offer = vm_os_offer(see accepted values in the table below);
```

Comment:
add a space before the bracket - _offer (see

Reply:
Done